core/src/main/scala/com/softwaremill/events/EventMachine.scala

```diff
@@ -45,6 +46,24 @@ class EventMachine(
       })
   }
 
+  def failOverFromStoredEventsTo(timeLimit: OffsetDateTime = OffsetDateTime.now()): Unit = {
```

Comment:
hm, I might not fully understand, do you want me to substitute default parameter `now()`? method needs to be parametrized cuz I want user to recover to some point in time and be default it might be now. do you mean `timeLimit: OffsetDateTime = clock.instant().atOffset(ZoneOffset.UTC)`?